cdk/apps/dracarys/dracarys/dracarys_stack.py

```diff
@@ -0,0 +1,61 @@
+from aws_cdk import (
+    aws_lambda,
+    aws_iam,
+    aws_s3,
+    aws_sqs,
+    aws_ssm,
+    aws_lambda_event_sources as lambda_event_source,
+    Stack,
+    Duration
+)
+
+from constructs import Construct
+
+class DracarysStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        lambda_role = aws_iam.Role(scope=self, id='dracarys-lambda-role',
+                                assumed_by =aws_iam.ServicePrincipal('lambda.amazonaws.com'),
+                                role_name='dracarys-lambda-role',
+                                managed_policies=[
+                                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                                    'service-role/AWSLambdaBasicExecutionRole'),
+                                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                                    'service-role/AWSLambdaVPCAccessExecutionRole'),
+                                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                                    'SecretsManagerReadWrite'),
```

Comment:
Why does it need write permissions?

Reply:
Couldn't find an AWS managed for SecretsManagerRead(Only) quick and didn't want to diy a non-managed policy, but definitely needs fixing :)